openstack/intel-worker/playbooks/roles/virtual_machine_create/tasks/ignition.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - name: Create worker.ign and store it to /var/www/html/ignition/
   ansible.builtin.get_url:
-    url: https://{{ bastion_ip }}:22623/config/worker
+    url: https://{{ virtual_machine_create_bastion_ip }}:22623/config/worker
```

Comment:
```suggestion
    url: https://{{ bastion_ip }}:22623/config/worker
```

Reply:
let's just call it bastion_ip

Reply:
Ansible lint expects to have a role name as a prefix for the variable names . if we just keep bastion_ip it was giving error .